spec/suites/layer/marker/MarkerSpec.js

```diff
@@ -237,17 +237,19 @@ describe("Marker", function () {
 			expect(mapSpy.called).not.to.be.ok();
 		});
 
-		it("do not catch event if it does not listen to it", function () {
+		it("do not catch event if it does not listen to it", function (done) {
 			var marker = new L.Marker([55, 37]);
 			map.addLayer(marker);
 			marker.once('mousemove', function (e) {
 				// It should be the marker coordinates
-				expect(e.latlng).to.be.nearLatLng(marker.getLatLng());
+				expect(e.latlng.equals(marker.getLatLng())).to.be.equal(true);
```

Comment:
Why `to.be.equal(true)` and not simply `to.be(true)`?
(And why not just `assert e.latlng == marker.getLagLng()`? Ah, we are not in python, sorry ;))